cfgov/v1/atomic_elements/organisms.py

```diff
@@ -911,6 +911,13 @@ class SnippetList(blocks.StructBlock):
     heading = blocks.CharBlock(required=False)
     body = blocks.RichTextBlock(required=False)
     image = atoms.ImageBasic(required=False)
+    actions_column_width = blocks.CharBlock(
```

Comment:
Can the `placeholder` attribute be set here too?
This isn't very useful:

![screen shot 2017-10-12 at 3 17 57 pm](https://user-images.githubusercontent.com/704760/31514724-95122558-af60-11e7-90bf-ac1c62c3f96e.png)